src/EventStore.Client/ResolvedEvent.cs

```diff
@@ -1,22 +1,56 @@
 #nullable enable
 namespace EventStore.Client {
+	/// <summary>
+	/// A structure representing a single event or a resolved link event.
+	/// </summary>
 	public readonly struct ResolvedEvent {
+		/// <summary>
+		/// If this <see cref="ResolvedEvent"/> represents a link event, the <see cref="OriginalEvent"/>
+		/// will be the link, otherwise it will be the event.
+		/// </summary>
 		public readonly EventRecord Event;
+
+		/// <summary>
+		/// The link event if this <see cref="ResolvedEvent"/> is a link event.
+		/// </summary>
 		public readonly EventRecord? Link;
 
+		/// <summary>
+		/// Returns the event that was read or which triggered the subscription.
+		///
+		/// If this <see cref="ResolvedEvent"/> represents a link event, the Link
+		/// will be the <see cref="OriginalEvent"/>, otherwise it will be the
+		/// event.
```

Comment:
```suggestion
		/// If this <see cref="ResolvedEvent"/> represents a link event, the <see cref="Link"/>
		/// will be the <see cref="OriginalEvent"/>, otherwise the <see cref="Event"/> will be the
		/// <see cref="OriginalEvent"/>
```